.readthedocs.yml

```
@@ -2,7 +2,7 @@ version: 2
 build:
   os: ubuntu-24.04
   tools:
-    python: "3.13"
+    python: "3.12" # dowhy breaks for Python 3.13https://github.com/py-why/dowhy/pull/1328
```

The PR has been merged and a release has been made.
```suggestion
    python: "3.13"
```

---

It might still not work but the comment is a bit outdated.